src/solution-template.js

```diff
@@ -23,23 +51,39 @@ let squaresLeft = 0;
 let bombProbability = 3;
 let maxProbability = 15;
 
-
-function minesweeperGameBootstrapper(rowCount, colCount) {
+function minesweeperGameBootstrapper(difficulty){
     let easy = {
         'rowCount': 9,
         'colCount': 9,
     };
     // TODO you can declare here the medium and expert difficulties
 
-    if (rowCount == null && colCount == null) {
+    let medium = {
+        'rowCount': 15,
+        'colCount': 15
+    }
+
+    let hard = {
+        'rowCount': 20,
+        'colCount': 20
+    }
+
+    if (difficulty == null) {
         // TODO have a default difficulty
+        rowCount = easy.rowCount;
+        colCount = easy.colCount;
     } else {
-        generateBoard({'rowCount': rowCount, 'colCount': colCount});
+        if(difficulty === 'easy')
+            generateBoard({ 'rowCount': easy.rowCount, 'colCount': easy.colCount });
+        else if(difficulty === 'medium')
+            generateBoard({ 'rowCount': medium.rowCount, 'colCount': medium.colCount });
+        else if(difficulty === 'hard')
+            generateBoard({ 'rowCount': medium.rowCount, 'colCount': medium.colCount });
```

Comment:
Un mic hint : pentru cand ai mai multe cazuri poti folosi switch, sau sa folosesti o abordare puțin mai moderna, cu object literals. Oricum e bine si cum ai facut tu, dar am vrut sa iti dau un sfat, sa te uiti pe object literals